engine/configurator/conf_cyaml.h

```diff
@@ -0,0 +1,323 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/** @file
+ * @brief YAML configuration types and schema
+ *
+ * API definitions.
+ *
+ * Copyright (C) 2018-2022 OKTET Labs Ltd. All rights reserved.
+ */
+
+#ifndef __TE_CONF_CYAML_H__
+#define __TE_CONF_CYAML_H__
+
+#include <cyaml/cyaml.h>
+#include "conf_api.h"
+#include "conf_types.h"
+#include "logger_api.h"
+
+/*
+ ******************************************************************************
+ * C data structure for storing a backup.
+ *
+ * This is what we want to load the YAML into.
+ ******************************************************************************
+ */
+
+/** Enumeration for object or instance. It answers if it is object */
+enum scope {
+    CFG_INSTANCE = 0, /* It is a default value */
+    CFG_OBJECT
```

Comment:
Enum members should have a prefix consistent with `enum` name, so if it's `enum cfg_cyaml_scope`, so the fields should be named `CFG_CYAML_SCOPE_INSTANCE` and `CFG_CYAML_SCOPE_OBJECT`.

Reply:
Fixed